README.md

```diff
@@ -5,9 +5,21 @@
 # Compass
 
 ## Overview
-A flexible and easy way to register, manage and group your Applications across Kyma Runtimes.
 
--   [Documentation](./docs/README.md)
+Compass (also known as Management Plane Services) is a multi-tenant system which consists of components that provide a way to register, group, and manage your applications across multiple Kyma runtimes. Using Compass, you can control and monitor your application landscape in one central place.
+
+Compass allows for registering different types of applications and runtimes.
+These are the types of possible integration levels between an application and Compass:
+- basic - administrator manually provides API/Events Metadata to Compass. This type of integration is used mainly for simple use-case scenarios and doesn't support all features.
+- application - integration with Compass is built-in inside the application.
+- proxy - a highly application-specific proxy component provides the integration.
+- service -  a central service provides the integration for a class of applications. It manages multiple instances of these applications. You can integrate multiple services to support different types of applications.
+
+You can register any runtime, providing that it fulfills a contract with Compass and implements its flow. First, your runtime must get a trusted connection to Compass. It must also allow for fetching application definitions and using these applications in a given tenant. The example runtimes are Kyma (Kubernetes), CloudFoundry, Serverless, etc.
```

Comment:
- Shouldn't we capitalize words like Runtime etc.?
- Maybe we should also link to Terminology?

Reply:
- We do not capitalize runtime in the MP docs. 
- There is a link to the documentation at the end of this section. I guess it's sufficient.